File: python/Bioskop.py
class Bioskop: #membuat class namanya Bioskop
    #untuk penggunaan (__) atau double underscore untuk membuat attribut tersebut menjadi private 
    def __init__(self, id_bioskop: str, nama_bioskop: str, alamat: str, snack_jual: str, jumlah_studio: int): 
        self.__id_bioskop = id_bioskop  #menyimpan ID bioskop
        self.__nama_bioskop = str(nama_bioskop)  #menyimpan nama bioskop 
        self.__alamat = str(alamat)  #menyimpan alamat bioskop
        self.__snack_jual = str(snack_jual)  #menyimpan snack yang dijual
        self.__jumlah_studio = int(jumlah_studio)  #menyimpan jumlah studio
    
    #setter untuk mengubah id
    def setId(self, id_bioskop: str):
        self.__id_bioskop = id_bioskop #

    #getter untuk mengambil id
    def getId(self) -> str:
        return self.__id_bioskop #mengembalikan nilai id
    
    #setter untuk mengubah nama
    def setNama (self, nama_bioskop: str):
        self.__nama_bioskop = nama_bioskop
    
    #getter untuk mengambil nama
    def getNama(self) -> str:
        return self.__nama_bioskop

    #setter untuk mengubah alamat
    def setAlamat(self, alamat: str):
        self.__alamat = alamat
    
    #getter untuk mengambil alamat
    def getAlamat(self) -> str:
        return self.__alamat
    
    #setter untuk mengubah snack yang dijual
    def setSnack(self, snack_jual: str):
        self.__snack_jual = snack_jual
    
    #getter untuk mengambil snack yang dijual
    def getSnack(self) -> str:
        return self.__snack_jual
    
    #setter untuk mengubah jumlah studio
    def setStudio(self, jumlah_studio: int):
        self.__jumlah_studio = jumlah_studio
    
    #getter untuk mengambil jumlah studio
    def getStudio(self) -> int:
        return self.__jumlah_studio
    
    def tampil(self) : #berfungsi untuk menampilkan data bioskopnya
        print("Id Bioskop      : ",self.__id_bioskop) #menampilkan id bioskop
        print("Nama Bioskop    : ",self.__nama_bioskop) #menampilkan nama bioskop
        print("Alamat Bioskop  : ",self.__alamat) #menampilkan alamat dari bioskop
        print("Snack Dijual    : ",self.__snack_jual) #menampilkan snack yang dijual di bioskop
        print("Jumlah Studio   : ",self.__jumlah_studio) #menampilkan jumlah studio
    
    

File: php/Bioskop.php
<?php
    class Bioskop{ //membuat class namanya itu Bioskop
        private string $id_bioskop; //menyimpan id bioskop
        private string $nama_bioskop; //untuk menyimpan nama bioskop
        private string $alamat; //menyimpan alamat
        private string $snack_jual; //menyimpan snack yang dijual
        private int $jumlah_studio; //menyimpan jumlah studio
        private string $gambar; //menyimpan path gambar

        public function __construct(string $id_bioskop, string $nama_bioskop, string $alamat, string $snack_jual, int $jumlah_studio, string $gambar){ //constructor untuk membuat object Bioskop
            $this->id_bioskop = $id_bioskop; //mengisi atribut ID dengan nilai dari parameter
            $this->nama_bioskop = $nama_bioskop; //mngisi atribut nama dengan nilai dari parameter
            $this->alamat = $alamat; //mwngisi atribut alamat dengan nilai dari parameter
            $this->snack_jual = $snack_jual; //mengisi atribut snack dengan nilai dari parameter
            $this->jumlah_studio = $jumlah_studio; //mngisi atribut jumlah studio dengan nilai dari parameter 
            $this->gambar = $gambar; //mengisi path gambar object
        }

        //setter untuk mengubah id
        public function setId($id_bioskop){
            $this -> id_bioskop = $id_bioskop; //mengubah nilai id
        }
        //getter untuk mengambil id
        public function getId(){
            return $this->id_bioskop; //mengembalikan nilai id
        }

        //setter untuk mengubah nama
        public function setNama ($nama_bioskop){
            $this -> nama_bioskop = $nama_bioskop; //mengubah nilai nama bioskop
        }
        //getter untuk mengambil nama
        public function getNama(){
            return $this->nama_bioskop; //mengembalikan nilai nama bioskop
        }

        //setter untuk mengubah alamat
        public function setAlamat($alamat){
            $this-> alamat = $alamat; //mengubah nilai alamat
        }
        //getter untuk mengambil alamat
        public function getAlamat(){
            return $this ->alamat; //mengembalikan nilai alamat
        }

        //setter untuk mengubah snack yang dijual
        public function setSnack($snack_jual){
            $this ->snack_jual = $snack_jual;
        }
        //getter untuk mengambil snack yang dijual
        public function getSnack(){
            return $this-> snack_jual; //mengembalikan nilai snack_jual
        }

        //setter untuk mengubah jumlah studio
        public function setStudio($jumlah_studio){
            $this -> jumlah_studio = $jumlah_studio; //mengubah nilai jumlah studio
        }
        //getter untuk mengambil jumlah studio
        public function getStudio(){
            return $this -> jumlah_studio; //mengembalikan nilai jumlah studio
        }

        //setter untuk mengubah gambar
        public function setGambar($gambar){
            $this -> gambar = $gambar; //mengubah nilai gambar
        }
        //getter untuk mengambil gambar
        public function getGambar(){
            return $this -> gambar; //mengembalikan nilai gambar
        }
    }
?>